website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -68,6 +68,17 @@ Your Consul configuration must meet the following requirements to use admin part
 
 - Consul 1.11.1 and newer
 
+### General Networking Requirements
+
+All Consul clients must be able to communicate with the Consul servers in the `default` partition. All servers must also be able to communicate with the clients.
```

Comment:
It might be the phrasing that is throwing me off, but my initial thought from reading this sentence is that it is not technically correct. It is true that client and server agents need to be able to communicate with each other, but the clients will communicate with the servers over the gossip pool for its assigned partition, not the `default` partition.